File: lista/zadanie2.c
#include <stdio.h>
#include <math.h>

void solve(float a, float b, float c)
{
	float delta = b*b - 4*a*c;

	if (delta < 0)
	{
		printf("Rozwiazania rownania nie naleza do zbioru liczb rzeczywistych\n");
	}
	else if (delta == 0)
	{
		printf("%f\n", ((-1 * b) / (2*a)));
	}
	else
	{
		printf("%f, %f", ((-1*b-sqrt(delta))/(2*a)), ((-1*b+sqrt(delta))/(2*a)));
	}

}

int main()
{
	float a;
	float b;
	float c;
	printf("Enter value a: ");
	scanf("%f", &a);
	printf("Enter value b: ");
	scanf("%f", &b);
	printf("Enter value c: ");
	scanf("%f", &c);

	solve(a, b, c);

	return 0;
}

File: list_1/zadanie2.py
import math

def main():
	def solve(a, b, c):
		delta = b*b - 4 * a * c
		if delta < 0:
			print("Rozwiazania rownania nie naleza do zbioru liczb rzeczywistych\n")
		elif delta == 0:
			print((-1*b) / (2*a))
		else:
			print((-1*b-math.sqrt(delta))/(2*a),(-1*b+math.sqrt(delta))/(2*a))
	
	a = int(input("Enter value a: "))
	b = int(input("Enter value b: "))
	c = int(input("Enter value c: "))

	solve(a, b, c)


if __name__ == "__main__":
	main()